models/country-extended-expanded.ts

```diff
@@ -70,7 +70,7 @@ export enum CountryExtendedExpanded {
   CongoDemocraticRepublicOfThe = "CONGO, DEMOCRATIC REPUBLIC OF THE",
   CookIslands = "COOK ISLANDS",
   CostaRica = "COSTA RICA",
-  CteDivoire = "CÔTE D'IVOIRE",
+  CteDivory = "CÔTE D`IVORY",
```

Comment:
It looks like Côte D'Ivoire is correct? https://en.wikipedia.org/wiki/Ivory_Coast